src/back-end/lib/server/adapters.ts

```diff
@@ -135,7 +135,7 @@ export function express<ParsedReqBody, ValidatedReqBody, ReqBodyErrors, HookStat
       const setSessionId = (id: string) => expressRes.cookie(SESSION_COOKIE_NAME, id, {
         signed: true,
         httpOnly: true,
-        sameSite: 'Lax',
+        sameSite: 'lax',
```

Comment:
I assumed this was a typo, so instead of setting `sameSite` to `any`, I changed the capitalization. If that seems wrong, I'll do the `any` fix.

Reply:
It was a type error being thrown with the capital L?

Reply:
`sameSite` is only allowed to be a few things, and 'lax' is allowed but 'Lax' isn't:
![Screenshot from 2022-01-11 13-57-41](https://user-images.githubusercontent.com/77306764/149027826-c5f82ca0-05cd-4fb4-865f-1633cb04f5ff.png)

Reply:
And actually, I didn't realize before but that's a node_modules file, so I shouldn't change it to `any`